apps/showcase/src/assets/trainings/sdk/steps/sdk-generation/instructions.html

```diff
@@ -0,0 +1,93 @@
+</p>
+<pre class="w-100 bg-body-tertiary px-5 pre-whitespace">
+  <code>
+<span class="text-success">?</span> Project name (NPM package scope, package.json name will be &#64;&#123;projectName&#125;/&#123;packageName&#125;)? <span class="text-primary">training-project</span>
+  </code>
+</pre>
+<p>
+  After setting the project name, you will see the shell of your SDK repository being generated with a structure that looks something like the following:
+</p>
+<ul>
+  <li><span class="fa-folder-open"></span> src
+    <ul>
+      <li><span class="fa-folder"></span> api</li>
+      <li><span class="fa-folder"></span> fixtures</li>
+      <li><span class="fa-folder"></span> helpers</li>
+      <li><span class="fa-folder-open"></span> models
+        <ul>
+          <li><span class="fa-folder"></span> base</li>
+          <li><span class="fa-folder"></span> core</li>
+          <li><span class="fa-folder"></span> custom</li>
+        </ul>
+      </li>
+      <li><span class="fa-folder"></span> spec</li>
+    </ul>
+  </li>
+</ul>
+<p>
+  You should also observe these scripts in the <code>package.json</code> file (with the correct project and package names):
+</p>
+<pre class="w-100 bg-body-tertiary px-5">
+  <code>
+"generate": "schematics @ama-sdk/schematics:typescript-core",
+"spec:regen": "npm run generate -- --generator-key training-project-training-sdk && amasdk-clear-index",
+"spec:upgrade": "npm run spec:regen",
+  </code>
+</pre>
+
+<h4>Generation of the API specifications</h4>
+<p>
+  The repository does not contain the specification file since we did not pass it as a parameter to the command, so it must be created manually.
+  Create a new file with your API specifications (created in the previous step). The standard convention is to name this file <code>open-api.yaml</code>.
+</p>
+<p>
+  <i>
+    If you have an existing specification file, you have the option to pass the parameter <code>--spec-path [path to your openapi file]</code> to the
+    create command used above, which will add your specification file to the repository.
+  </i>
+</p>
+<p>
+  Next, specify the path of your specification file in the <code>inputSpec</code> property of the <code>openapitools.json</code> file at the root.
+</p>
+<p>
+  You can now run the command <code>yarn spec:upgrade</code> and notice that the <b>api</b> and <b>models/base</b> folders have been updated with the paths and definitions from your specification file.
+</p>
+<p>
+  <i>
+    <b>NOTE:</b> Notice that the <code>yarn spec:upgrade</code> command is equivalent to running the command below, which means that we use the configuration of the
+    <code>openapitools.json</code> file. These configuration properties can also be passed as parameters to the command line (to set or override the property).
+    This tip will be useful for the next steps of the training.
+  </i>
+</p>
+<pre class="w-100 bg-body-tertiary px-5 pre-whitespace">
+  <code>
```

Comment:
should not be tagged with language?
You can not use `o3r-copy-text-pres` instead?

(same question for other `code`)

Reply:
this is not possible with the current setup, the html is just used in an `[innerHTML]` with no angular compilation
however, I am currently working on replacing that with `md` files using `ngx-markdown` which include the code highlighting using `prismjs` (check draft PR https://github.com/AmadeusITGroup/otter/pull/2414)
I am also planning on replacing the usage  `o3r-copy-text-pres` to `marked` so we only use one code highlighting technology (the DF demo site also uses PrismJS)